evaluate/metrics/metrics_test.go

```diff
@@ -1,12 +1,55 @@
 package metrics
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/zimmski/osutil/bytesutil"
 )
 
+func TestMetricsString(t *testing.T) {
+	type testCase struct {
+		Name string
+
+		Metrics Metrics
+
+		ExpectedString string
+	}
+
+	validate := func(t *testing.T, tc *testCase) {
+		t.Run(tc.Name, func(t *testing.T) {
+			actualString := tc.Metrics.String()
+
+			assert.Equal(t, tc.ExpectedString, actualString)
+		})
+	}
+
+	validate(t, &testCase{
+		Name: "Empty Metrics",
+
+		Metrics: Metrics{},
+
+		ExpectedString: "files-executed=0.0%(0/0), files-problems=0.0%(0/0), coverage-statement=0.0%(0/0), no-excess-response=0.0%(0/0)",
+	})
+
+	validate(t, &testCase{
+		Name: "Empty Metrics",
+
+		Metrics: Metrics{
+			Total: 4,
+			Assessments: Assessments{
+				AssessmentKeyCoverageStatement: 1,
+				AssessmentKeyFilesExecuted:     2,
+				AssessmentKeyFilesProblems:     3,
+				AssessmentKeyNoExcessResponse:  4,
+			},
+		},
+
+		ExpectedString: "files-executed=50.0%(2/4), files-problems=75.0%(3/4), coverage-statement=25.0%(1/4), no-excess-response=100.0%(4/4)",
```

Comment:
The percentage doesn't make sense. now that i look at this? let's drop it. what we want is a "base" where we then recieve the percentages from. For now. good enough to just print the absolute values.

Reply:
should we then also drop tracking the total count? because we really only need it for the percentage computation.

Reply:
alright... question answered below